server/service/integration_live_queries_test.go

```diff
@@ -31,7 +31,7 @@ import (
 
 func TestIntegrationLiveQueriesTestSuite(t *testing.T) {
 	testingSuite := new(liveQueriesTestSuite)
-	testingSuite.s = &testingSuite.Suite
+	testingSuite.withServer.s = &testingSuite.Suite
```

Comment:
This change is just for me -- my IDE doesn't like the confusion between testingSuite.Suite.s and testingSuite.withServer.s